### ISSUE TITLE
Push notifications and targets

### ISSUE THREAD

**Issue by pbailey64** [jiscdev/la-app#47]
Several insts have requested that they could push targets to students "e.g. spend 2 hours in the lab this week". It would also be useful to push other messages to the app e.g. positive messages "well done on achieving your targets for last week". Linked to this is a request to allow "groups" to be created by students or tutors (rather than just course groups). This would be useful for group assignments to see contributions. 
Inclusion of the tutor in a group so they could see and share. Tutors are also requesting that they can see the same data as the student gets in the app (this may be better done via  staff portal).

**Comment by robwynj** [comment]
This will go into version 2 - requires a design sprint with @michaelwebbjisc, @pbailey64 and @TherapyBox.

**Comment by TherapyBox** [comment]
Working on this.

Just want to check: should it be identified who sent the target in the first place? Do you  want a student to have the option to confirm or reject a target before it is sent over?

We don't have any plan for study groups at the moment but could resolve if required.

**Comment by pbailey64** [comment]
Yes - they should know which tutor sent the target and also be able to accept or reject it into their targets.

Lets leave the study groups and group targets to a future release.
